LICENSE.txt

```diff
@@ -1,21 +1,21 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Neil McNab, Hampus Wessman
+Copyright (c) 2007-2024 Neil McNab, Hampus Wessman, and other contributors
```

Comment:
We need a set of designated copyright holders to make usage decisions, if we open this up we make it impossible to make decisions on licensing without reviewing with all previous contributers

Reply:
Ah, understood. Would it make sense to add those who have contributed to date, or do we want to leave it to Neil and Bram?

I've also seen people listed for different years, so we could technically add yourself from when you adopted the project?

Reply:
Adding people makes sense, we cna just have a contributors list and mention those poeple in the copyright statement, that would be fine. Legally it's probably very hard to work with "go look at git to see who contributed to figure out who to contact for copyright issues" :wink: